scene/gui/tree.cpp

```diff
@@ -4452,17 +4452,25 @@ void Tree::update_scrollbars() {
 	const int title_button_height = _get_title_button_height();
 
 	Size2 tree_content_size = content_rect.get_size() - Vector2(0, title_button_height);
-	bool display_vscroll = internal_min_size.height > tree_content_size.height;
-	bool display_hscroll = internal_min_size.width > tree_content_size.width;
+
+	bool display_vscroll = v_scroll_enabled && 
+			internal_min_size.height > tree_content_size.height;
```

Comment:
```suggestion
	bool display_vscroll = v_scroll_enabled && internal_min_size.height > tree_content_size.height;
```
This looks better in a single line tbh.
Same below. These conditions aren't that long, making them broken impairs readability.